analyses/org.eclipse.tracecompass.incubator.executioncomparision.core/src/org/eclipse/tracecompass/incubator/internal/executioncomparision/core/DifferentialCallGraphProvider.java

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (c) 2023 École Polytechnique de Montréal
```

Comment:
All of the copyrights may be updated to 2024

Reply:
Done